### ISSUE TITLE
Hovering over the body doesnt announce combo field

### ISSUE THREAD

**Issue by ivaynberg** [select2/select2#5567]
Hovering over the white space in between the textfield and the right edge of the component does not announce the textfield, but clicking that area focuses the textfield. Maybe add a label to it?

**Comment by ivaynberg** [comment]
this is not valid, only Edge reads out on hover, so this is not a standard behavior we need to worry aobut.
